lib/virtual.js

```diff
@@ -12,7 +12,9 @@ function VirtualModulesPlugin(compiler) {
 		var originalPurge = compiler.inputFileSystem.purge;
 
 		compiler.inputFileSystem.purge = function() {
-			originalPurge.call(this, arguments);
+			if (originalPurge) {
```

Comment:
I'm gonna merge that from your original PR on svelte-loader, so that refs remains OK if my code eventually get merged there.